src/java.base/macosx/native/libjli/java_md_macosx.m

```diff
@@ -60,115 +60,52 @@
- * virtual machine; they must be screened out by the launcher.
+ *  - JLI_Launch then ultimately calls JVMInit.
  *
- * The version specification (if any) is processed first by the
- * platform independent routine SelectVersion.  This may result in
- * the exec of the specified launcher version.
+ *  - JVMInit then invokes JavaMain.
  *
- * Now, in most cases,the launcher will dlopen the target libjvm.so. All
- * required libraries are loaded by the runtime linker, using the known paths
- * baked into the shared libraries at compile time. Therefore,
- * in most cases, the launcher will only exec, if the data models are
- * mismatched, and will not set any environment variables, regardless of the
- * data models.
+ *  - JavaMain, before launching the application, invokes PostJVMInit.
  *
+ *  - PostJVMInit invokes ShowSplashScreen which displays a splash screen for the application, if applicable.
  *
+ *  - Control then returns back from PostJVMInit into JavaMain, which then loads the application's main class
+ *    and invokes the relevant main() Java method.
  *
- *  Main
- *  (incoming argv)
- *  |
- * \|/
- * CreateExecutionEnvironment
- * (determines desired data model)
- *  |
- *  |
- * \|/
- *  Have Desired Model ? --> NO --> Is Dual-Mode ? --> NO --> Exit(with error)
- *  |                                          |
- *  |                                          |
- *  |                                         \|/
- *  |                                         YES
- *  |                                          |
- *  |                                          |
- *  |                                         \|/
- *  |                                CheckJvmType
- *  |                               (removes -client, -server etc.)
- *  |                                          |
- *  |                                          |
- * \|/                                        \|/
- * YES                             Find the desired executable/library
- *  |                                          |
- *  |                                          |
- * \|/                                        \|/
- * CheckJvmType                             POINT A
- * (removes -client, -server, etc.)
- *  |
- *  |
- * \|/
- * TranslateDashJArgs...
- * (Prepare to pass args to vm)
- *  |
- *  |
- * \|/
- * ParseArguments
- * (processes version options,
- *  creates argument list for vm,
- *  etc.)
- *   |
- *   |
- *  \|/
- * POINT A
- *   |
- *   |
- *  \|/
- * Path is desired JRE ? YES --> Continue
- *  NO
- *   |
- *   |
- *  \|/
- * Paths have well known
- * jvm paths ?       --> NO --> Continue
- *  YES
- *   |
- *   |
- *  \|/
- *  Does libjvm.so exist
- *  in any of them ? --> NO --> Continue
- *   YES
- *   |
- *   |
- *  \|/
- * Re-exec / Spawn
- *   |
- *   |
- *  \|/
- * Main
+ *  - JavaMain then returns back an integer result which then gets propagated as a return value all the way out
+ *    of the JLI_Launch function.
```

Comment:
Are you going to re-flow this comment block o reduce the line lengths, the really long lines are real bitlly annoying for side-by-side diffs and also inconsistent with the rest of the file.

Reply:
Hello Alan, I just pushed a update to the PR to reduce the line length of these comments. I'm hoping this reads better.